homeassistant/components/minecraft_server/sensor.py

```diff
@@ -137,24 +136,26 @@ class MinecraftServerSensorEntity(MinecraftServerEntity, SensorEntity):
 
     def __init__(
         self,
-        server: MinecraftServer,
+        coordinator: MinecraftServerCoordinator,
         description: MinecraftServerSensorEntityDescription,
     ) -> None:
         """Initialize sensor base entity."""
-        super().__init__(server)
+        super().__init__(coordinator)
         self.entity_description = description
-        self._attr_unique_id = f"{server.unique_id}-{description.key}"
+        self._attr_unique_id = f"{coordinator.unique_id}-{description.key}"
+        self._attr_native_value = description.value_fn(coordinator.data)
 
-    @property
-    def available(self) -> bool:
-        """Return sensor availability."""
-        return self._server.online
+        if func := description.attributes_fn:
+            self._attr_extra_state_attributes = func(coordinator.data)
```

Comment:
```suggestion
        self._update_attr()

    @callback
    def _update_attr(self) -> None:
        """Update _attr."""
        self._attr_native_value = description.value_fn(coordinator.data)

        if func := description.attributes_fn:
            self._attr_extra_state_attributes = func(coordinator.data)
```

So we only have one place where we update the state/attributes.

Reply:
I was thinking about it but then decided against it, because of the implementation of other similar integrations 👍